src/headers/themes.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef THEMES_H
+#define THEMES_H
+
+#include "color.hpp"
+#include "global.hpp"
+#include <cmath>
+#include <limits>
+#include <vector>
+
+class Themes {
+public:
+  Themes()
+      : fibonacci_numbers{1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 377, 610},
+        elements{"H", "He", "Li", "Be", "B",  "C", "N",
+                 "O", "F",  "Ne", "Na", "Ca", "Sc"},
+        programming_languages{"C",   "C++", "PHP",  "C#", "Py", "Bash", "Java",
+                              "SQL", "CSS", "HTML", "JS", "Go", "Rust"} {}
```

Comment:
Ask the question: "If I had `400` themes, is the `[Theme's class constructor]` the only way a theme can be added to this game?"
* It is possible that if it is not sustainable to do so, you may have to make some API to accept:
 `[some "x theme" datatype]` and add it to `[a list of some "x theme" datatypes]`

Reply:
Changed: `Theme` defines just one theme datatype, and `ThemeController` handles the list of `Theme` objects